modules/images-configuration-allowed.adoc

```diff
@@ -64,11 +64,40 @@ The Machine Config Operator (MCO) watches the `image.config.openshift.io/cluster
 ifndef::openshift-rosa,openshift-dedicated[]
 .Verification
 
-* To check that the registries have been added to the policy file, use the following command on a node:
+* To check that the registries have been added to the policy file, enter the following command to obtain a list of your nodes:
 +
 [source,terminal]
 ----
-$ cat /host/etc/containers/policy.json
+$ oc get node
```

Comment:
```suggestion
$ oc get nodes
```

Reply:
@jab-rh Is there a different in running `oc get node` vs. `oc get nodes`? I tested this with `oc get node`, eg: 

stevsmit@stevsmit ~ $ oc get node

NAME                         STATUS                     ROLES    AGE   VERSION
ip-10-0-1-107.ec2.internal   Ready,SchedulingDisabled   worker   53m   v1.27.8+4fab27b
ip-10-0-1-253.ec2.internal   Ready

Reply:
@stevsmit, I think both work, it's an API alias, but usually in commands I see the plural form used, just for consistency.